ami_environment.yml

```diff
@@ -15,3 +15,5 @@ dependencies:
   - torchvision-cpu
   - pip:
     - https://api.github.com/repos/paris-saclay-cds/ramp-workflow/zipball/master
+    - tensorflow
+    - keras
```

Comment:
Please move these two above. Keras and tensorflow are in conda-forge, no need to pip install them.